docs/QUICK_START.md

```diff
@@ -125,38 +125,58 @@ Below is a fragment of a NixOS configuration that enables the Secure
 Boot stack.
 
 ```nix
-nixosConfigurations = {
-  yourHost = nixpkgs.lib.nixosSystem {
-    system = "x86_64-linux";
+{
+  description = "A SecureBoot-enabled NixOS configurations";
 
-    modules = [
-      # ... other modules ...
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
 
-      lanzaboote.nixosModules.lanzaboote
+    # ...
```

Comment:
To me these placeholders do not add value, but instead make it slightly more annoying to copy and paste stuff.

Reply:
Fixed.